java/ql/src/experimental/Security/CWE/CWE-600/UncaughtServletException.qhelp

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>
+Even though the signatures for methods in a servlet include throws IOException, ServletException, it's a bad idea to let such exceptions be thrown. Failure to catch exceptions in a servlet could leave a system in a vulnerable state, possibly resulting in denial-of-service attacks, or the exposure of sensitive information because when a servlet throws an exception, the servlet container typically sends debugging information back to the user. And that information could be very valuable to an attacker.
```

Comment:
>throws IOException, ServletException

Maybe it would be good to format this as code to make it more clear that this is part of the signature

Reply:
Thanks @Marcono1234, I've formatted it as code to make it more clear.

Reply:
I am not completely sure, but I think you might have to use `<code>...</code>` here.

Reply:
Sorry @Marcono1234 . Yes, the HTML tag `<code>` shall be used here instead of markdown code. I've fixed the issue.